commodore/cluster.py

```diff
@@ -67,6 +69,18 @@ def update_target(cfg: Config, target):
     os.makedirs(file.parent, exist_ok=True)
     yaml_dump(render_target(target, cfg.get_components().keys()), file)
 
+    # TODO Remove before merge!
+    common = P('inventory/classes/common.yml')
+    os.makedirs(common.parent, exist_ok=True)
+    yaml_dump({
+        'classes': [
+            'global.distribution.${facts:distribution}',
+            'global.cloud.${facts:cloud}',
+            'global.cloud.${facts:cloud}.${facts:region}',
+            '${cluster:tenant}.${cluster:name}',
+
```

Comment:
```suggestion
            'global.lieutenant-instance.${facts:lieutenant-instance}',
```

Reply:
For some reason the `lieutenant-instance` class is not properly included if I try to compile the `c-old-fire-5788` cluster.

Reply:
Hmm, seems like the referencing breaks because the order of import changed: As mentioned above, `global.common` needs to be included before `common` for it to work properly.

Reply:
I did some tests and I fear we have to split the hierarchy definition and parameters into two files.
Currently both is defined in [common.yml](https://git.vshn.net/syn/commodore-defaults/-/blob/dynamic_hierarchy/common.yml). But this has the effect, that references used in there (e.g. `VAULT_ADDR: ${secret_management:vault_addr}`) are not properly overwritten, because they are evaluated first, before including other classes.

We could for example leave only the class imports in `common.yml` and move all the `parameters` definitions into a new file (e.g. `global.params`) which needs to be included as the first class.